make error : no new variables on left side of :=

I set this in my .zhsrc to avoid io time out
```sh
export GOPROXY=https://goproxy.io
```

Then I run the make, here is the output:
```sh
CGO_CXXFLAGS=" -std=c++11" go build -tags " rocksdb" -o bin/go-ycsb cmd/go-ycsb/*
# github.com/pingcap/go-ycsb/db/rocksdb
db/rocksdb/db.go:239:11: no new variables on left side of :=
Makefile:23: recipe for target 'build' failed
make: *** [build] Error 2
```

--- a/db/foundationdb/db.go
+++ b/db/foundationdb/db.go
@@ -153,7 +153,7 @@ func (db *fDB) Update(ctx context.Context, table string, key string, values map[
 		buf := db.bufPool.Get()
 		defer db.bufPool.Put(buf)
 
-		buf, err := db.r.Encode(buf, data)
+		buf, err = db.r.Encode(buf, data)
 		if err != nil {
 			return nil, err
 		}

--- a/db/rocksdb/db.go
+++ b/db/rocksdb/db.go
@@ -232,7 +232,7 @@ func (db *rocksDB) Update(ctx context.Context, table string, key string, values 
 	buf := db.bufPool.Get()
 	defer db.bufPool.Put(buf)
 
-	buf, err := db.r.Encode(buf, m)
+	buf, err = db.r.Encode(buf, m)
 	if err != nil {
 		return err
 	}

